docs/self-hosted-apps/git.md

```diff
@@ -0,0 +1,7 @@
+# Git
+
+As always, check out [awesome-selfhosted](https://github.com/awesome-selfhosted/awesome-selfhosted#project-management) but here are our favourites.
```

Comment:
This sentence appears too many times and as we explain in the main PR thread we're not looking to be a list of apps.